DATA605/README.md

```diff
@@ -0,0 +1,224 @@
+
+## Choosing a project
+
+- Each student should pick one of the projects from the [signup sheet](https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw/edit?gid=0#gid=0)
+  - The difficulty of the project does affect the final grade, but we want to
+    give a way for everyone to select a project based on their level of computer
+    literacy
+  - You will need to give us some information including your GitHub name so we
+    can add you to the repo
+
+- The project is individual
+  - Students can discuss and help each other (they will do that even if we say
+    not to)
+  - Students should not have exactly the same project
+  - If there are no more projects left for any reason, then we will add more
+
+- The goal is to get your hands dirty and figure things out
+  - Often working is all about trying different approaches until one works out
+  - Google and ChatGPT are your friends, but don't abuse them: copy-pasting
+    without understanding doesn't help you
+  - Make sure you understand what, how, and why a piece of code does
+ 
+- On the sign-up sheet, you will find several baseline project proposals
+  suggested by us. You can either choose from the given projects or propose your
+  own project. You can improve, change, and add other technologies and components
+  to enhance your project
+- Your project choice should align with your learning goals and interests,
+  presenting an excellent opportunity to explore various technologies and enhance
+  your resume.
+
+
+- If you choose one of the projects from the sign-up sheet, fill out the
+  corresponding information promptly. If you decide to propose a modification,
+  send us an email with the desired information and we will update the sign-up
+  sheet and Google Doc for you.
+
+- Please note that you are required to finalize your project selection within
+  one week
+- The project duration is approximately four weeks, so timely selection is
+  crucial for effective planning and execution.
+
+- Your grades will be influenced by factors such as project complexity, your
+  efforts and understanding, and adherence to given project guidelines.
+
+## Working
+
+- You will work in the same way open-source developers (and specifically
+  developers on Sorrentum) contribute to a project
+
+- Each step of the project is delivered by committing code to your dir
+  corresponding to your project (more below) and doing a GitHub Pull Request
+  (PR)
+  - You can / should commit regularly and not only once at the end
+  - This allows us to review intermediate results and give you feedback (like
+    companies adopting an Agile methodology do)
+- We will do a review of the project in the middle of the project and give you
+  some feedback on what to improve
+
+- You can model your working setup (TODO Krishna: Fill the details after confirmation)
+
+## Pre-requisites
+
+- Watch, star, and fork the Sorrentum repo (TODO Krishna: Correct this based on the input)
```

Comment:
We should make an assignment for people to do this... otherwise people don't do it, and we want to promote the project on GitHub